js/e2e.bats

```diff
@@ -1,7 +1,44 @@
 #!/usr/bin/env bats
 
+@test "should return valid digest for kubectl image" {
+    local image="ghcr.io/kubewarden/kubectl:v1.31.0"
+    local expected_digest=$(crane digest "$image")
+    echo "Expected digest = $expected_digest"
```

Comment:
yeah, all of that can be removed, it's useless because we're using a recorded session